Sources/Specs/GameSpec.swift

```diff
@@ -0,0 +1,60 @@
+import SnowManLogic
+import Swiftest
+
+class GameSpec: Swiftest.Spec {
+	let spec = describe("game logic") {
+		var guessManager: GuessManager!
+		var word: String!
+		var game: Game!
+		before() {
+			guessManager =  GuessManager(totalIncorrectGuessesAllowed: 5)
+			word = "apple"
+			game = Game(word: word, guessManager: guessManager)
+		}
+
+		describe("#checkForGuessInWord") {
+			var guess: Character!
+			before() {
+				guess = "a"
+			}
```

Comment:
the character variable and it's assignment in the before block are not necessary. It's only used in these 2 tests, and the values are different each time. Just bind the variables in the test case.